Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -782,102 +783,187 @@ void ACesium3DTileset::MarkTilesetDirty() {
   UE_LOG(LogCesium, Verbose, TEXT("Tileset marked Dirty"));
 }
 
-std::optional<ACesium3DTileset::UnrealCameraParameters>
-ACesium3DTileset::GetCamera() const {
-  std::optional<UnrealCameraParameters> camera = this->GetPlayerCamera();
+std::vector<ACesium3DTileset::UnrealCameraParameters>
+ACesium3DTileset::GetCameras() const {
+  std::vector<UnrealCameraParameters> cameras = this->GetPlayerCameras();
 
 #if WITH_EDITOR
-  if (!camera) {
-    camera = this->GetEditorCamera();
-  }
+  std::vector<UnrealCameraParameters> editorCameras = this->GetEditorCameras();
+  cameras.insert(
+      cameras.end(),
+      std::make_move_iterator(editorCameras.begin()),
+      std::make_move_iterator(editorCameras.end()));
 #endif
 
-  return camera;
+  return cameras;
 }
 
-std::optional<ACesium3DTileset::UnrealCameraParameters>
-ACesium3DTileset::GetPlayerCamera() const {
+std::vector<ACesium3DTileset::UnrealCameraParameters>
+ACesium3DTileset::GetPlayerCameras() const {
   UWorld* pWorld = this->GetWorld();
   if (!pWorld) {
-    return std::optional<UnrealCameraParameters>();
+    return {};
   }
 
-  APlayerController* pPlayerController = pWorld->GetFirstPlayerController();
-  if (!pPlayerController) {
-    return std::optional<UnrealCameraParameters>();
-  }
+  std::vector<ACesium3DTileset::UnrealCameraParameters> cameras;
 
-  APlayerCameraManager* pCameraManager = pPlayerController->PlayerCameraManager;
-  if (!pCameraManager) {
-    return std::optional<UnrealCameraParameters>();
-  }
+  for (auto playerControllerIt = pWorld->GetPlayerControllerIterator();
+       playerControllerIt;
+       playerControllerIt++) {
 
-  UGameViewportClient* pViewport = pWorld->GetGameViewport();
-  if (!pViewport) {
-    return std::optional<UnrealCameraParameters>();
-  }
+    const TWeakObjectPtr<APlayerController> pPlayerController =
+        *playerControllerIt;
+    if (pPlayerController == nullptr) {
+      continue;
+    }
 
-  FVector location;
-  FRotator rotation;
-  pCameraManager->GetCameraViewPoint(location, rotation);
-  double fov = pCameraManager->GetFOVAngle();
+    APlayerCameraManager* pCameraManager =
+        pPlayerController->PlayerCameraManager;
+    if (!pCameraManager) {
+      continue;
+    }
 
-  FVector2D size;
-  pViewport->GetViewportSize(size);
+    bool isOrthographic = pCameraManager->IsOrthographic();
+    double orthographicWidth = pCameraManager->GetOrthoWidth();
 
-  if (size.X < 1.0 || size.Y < 1.0) {
-    return std::nullopt;
+    UGameViewportClient* pViewport = pWorld->GetGameViewport();
+    if (!pViewport) {
+      continue;
+    }
+
+    FVector location;
+    FRotator rotation;
+    pCameraManager->GetCameraViewPoint(location, rotation);
+    double fov = pCameraManager->GetFOVAngle();
+
+    FVector2D size;
+    pViewport->GetViewportSize(size);
+
+    if (size.X < 1.0 || size.Y < 1.0) {
+      continue;
+    }
+
```

Comment:
There are several `if (....) continue;` blocks above. 
Are these anticipated cases? 
If so, a comment like `// This happens when (PIE, loading a level, switching views, ... whatever)` could be helpful. 
If not, may they be worth a warning like `LOG(Warning, "Whoopsie, why is there no CameraManager"`)`?